Backend/DTO/LoggingAPI/LoggingRequests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
What is the point of these DTOs?

Reply:
Is this the object we will be receiving from the Frontend?

Reply:
yes